code/src/java/pcgen/cdom/facet/analysis/ResultFacet.java

```diff
@@ -57,14 +57,19 @@ public Object getGlobalVariable(CharID id, String varName)
 
 	public Object getLocalVariable(CharID id, CDOMObject cdo, String varName)
 	{
+		String localScopeName = cdo.getLocalScopeName();
+		if(localScopeName == null){
+			return getGlobalVariable(id, varName);
+		}
+		
 		/*
 		 * TODO Given who this serves, this is an interesting situation. This
```

Comment:
Please delete the TODO, since you basically addressed it by dropping to Global when necessary.